requires-dev.txt

```diff
@@ -1,14 +1,14 @@
 dash_flow_example==0.0.5
 dash-dangerously-set-inner-html
 isort==4.3.21
-mock==4.0.1;python_version>="3.0"
+mock==4.0.2;python_version>="3.0"
 mock==3.0.5;python_version=="2.7"
-flake8==3.7.9
-PyYAML==5.3
+flake8==3.8.3
+PyYAML==5.3.1
 pylint==1.9.4;python_version<"3.7"
-pylint==2.3.1;python_version=="3.7"
-astroid==2.2.5;python_version=="3.7"
+pylint==2.5.3;python_version=="3.7"
```

Comment:
Updating pylint and flake8 has side-effects on other core/ancillary component libraries.